kubernetes/samples/scripts/create-weblogic-domain/model-in-image/README.md

```diff
@@ -59,218 +47,120 @@ It is helpful to understand the following high-level flow before running the sam
+1. The JAVA_HOME environment variable must be set and must reference a valid JDK8 installation.
+
+2. Setup a source directory env variable `SRCDIR`, an empty test directory, and a test directory environment variable `WORKDIR`.  Specifically:
 
-2. Setup a test directory, a source directory env variable, and a test directory environment variable.
+   - Store the location of the Operator source code in an environment variable `SRCDIR`.  For example:
 
-   - Store the location of the Operator source code in an environment variable `SRCDIR`.
+     ```
+     mkdir ~/wlopsrc
+     cd ~/wlopsrc
+     git clone https://github.com/oracle/weblogic-kubernetes-operator.git
+     export SRCDIR=$(pwd)/weblogic-kubernetes-operator
+     ```
+
+     For additional information about obtaining the operator source, see the [Developer Guide Requirements](https://oracle.github.io/weblogic-kubernetes-operator/developerguide/requirements/).
    
-   - Create an empty temporary working directory with 10g of space, and store its location in `WORKDIR`.
+   - Create an empty temporary working directory with 10g of space, and store its location in `WORKDIR`. For example:
 
-   ```
-   cd <top of source tree - should end with '/weblogic-kubernetes-operator'>
-   export SRCDIR=$(pwd)
-   cd <location of empty temporary directory with 10g of space>
-   export WORKDIR=$(pwd)
-   ```
+     ```
+     cd <location of empty temporary directory with 10g of space>
+     export WORKDIR=$(pwd)
+     ```
 
-3. Copy all the files in this sample to the working directory (substitute your Operator source location for SRCDIR).
+3. Copy all the files in this sample to the `$WORKDIR` working directory. For example:
 
    ```
    cp -R ${SRCDIR?}/kubernetes/samples/scripts/create-weblogic-domain/model-in-image/* ${WORKDIR?}
    ```
 
-4. Deploy the Operator and setup the Operaator to manage namespace `sample-domain1-ns`. 
+4. Deploy the Operator and setup the Operator to manage namespace `sample-domain1-ns`. 
+
    - For example, see [Quick Start](https://oracle.github.io/weblogic-kubernetes-operator/quickstart/) up through the `PREPARE FOR A DOMAIN` step. Note that you can skip the Quick Start steps for obtaining a WebLogic image and for configuring Traefik load balancer - as instead we we will generate our own image and setup an nginx load balancer instead.
-   - If you've already deployed the Operater, you can use `helm get values my-operator-release` to check if it is 
-   managing namespace `sample-domain1-ns` and use helm upgrade to add this namespace if needed.  For example:
+   - After you've deployed the Operater, check if it is managing namespace `sample-domain1-ns` and use helm upgrade to add this namespace if needed.  For example:
      ```
-     # get the helm release name for the running operator
+     # get the helm release name for the running operator, and use
+     # this release name in place of 'my-operator-release' below
```

Comment:
Do we want to specify this is helm 2 syntax only ?

Reply:
can you supply the helm 3 syntax?

Reply:
or merge the pull, and make the update yourself?